server/routes/request.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const {
+  getRequests,
+  createRequest,
+  updateRequestStatus,
+} = require("../controllers/request");
+
+router.route("/").get(protect, getRequests);
+router.route("/").post(protect, createRequest);
+router.route("/:requestId").put(protect, updateRequestStatus);
```

Comment:
Though `PUT` works for update, it's a good practice to use `PATCH` when you're not updating the whole resource :+1: